monkey/monkey_island/cc/ui/src/components/pages/ReportPage.js

```diff
@@ -675,6 +680,28 @@ class ReportPageComponent extends AuthComponent {
     );
   }
 
+  generateVsftpdBackdoorIssue(issue) {
+    return (
+      <li>
+        Update your VSFTPD server to the latest version vsftpd-3.0.3.
```

Comment:
It might not be the latest version when the user runs this

Reply:
The last release was around July 2015, I guess the wont update it anytime soon.